test/algorithms/ground_state_solvers/test_adapt_vqe.py

```diff
@@ -84,8 +86,10 @@ def get_solver(self, transformation):
                 qubit_mapping = transformation.qubit_mapping
                 two_qubit_reduction = transformation.molecule_info['two_qubit_reduction']
                 z2_symmetries = transformation.molecule_info['z2_symmetries']
-                initial_state = HartreeFock(num_orbitals, num_particles, qubit_mapping,
-                                            two_qubit_reduction, z2_symmetries.sq_list)
+
+                mapper = ParityMapper()
+                converter = QubitConverter(mappers=mapper)
```

Comment:
Just a comment. Given the operators need to be the same type, and we have changed the 2ndQop so that its not a container of different types we may well refactor this to just be a single mapper that is passed. When we have a MixedOperator later that could be the MixedMapper that is a contained too for individual mappers, so we still need one mapper at this level. Anyway by using this as a named param of QubitConvertor, rather than positional, this will need to be refactored to accomodate that. I guess we have a mix of named and not named here, for instance HF is done by position.